src/azure-cli-core/azure/cli/core/tests/test_cloud.py

```diff
@@ -157,37 +157,6 @@ def test_get_active_cloud_name_default(self):
         actual = get_active_cloud_name(cli)
         self.assertEqual(expected, actual)
 
-    def test_known_cloud_missing_endpoint(self):
```

Comment:
Is there a reason you removed this test?

Reply:
That test was replaced by test_modify_known_cloud() below but it was left in in the knack branch.